### ISSUE TITLE
Make the web HTTPS

### ISSUE THREAD

**Issue by Acontreras2k** [manulopez14/Proyecto-Pagina-Web#7]
I think It'd be a good idea to make our website secure. It doesn't really matter, but only being http makes the web insecure

**Comment by luciluci8** [comment]
I agree, the question is how to turn it into a HTTPS web.

**Comment by manulopez14** [comment]
I agree, we should work upon making our web secure. We will fix it in the next weeks.
